include/execution.hpp

```diff
@@ -4279,15 +4279,15 @@ namespace std::this_thread {
         nothrow_tag_invocable<sync_wait_with_variant_t, _Sender>) {
         return tag_invoke(sync_wait_with_variant_t{}, (_Sender&&) __sndr);
       }
-      template <execution::__single_value_variant_sender<__impl::__env> _Sender>
+      template <execution::sender _Sender>
```

Comment:
Maybe?

```suggestion
      template <execution::sender<__impl::__env> _Sender>
```

I'm on my phone, I'll need to double check this. Also, if you change this constraint here, do you need to change it for the other overloads?